src/client/app/containers/app.ts

```diff
@@ -18,8 +19,9 @@ import { Strings, environment } from '../helpers';
                     <command icon="OpenPaneMirrored" title="${Strings.runSideBySide}" (click)="runSideBySide()"></command>
                 </command>
                 <command [hidden]="isEmpty" icon="Share" [async]="sharing$|async" title="${Strings.share}">
-                    <command icon="PageCheckedin" title="${Strings.shareMenuPublic}" (click)="shareGist(true)"></command>
-                    <command icon="ProtectedDocument" title="${Strings.shareMenuPrivate}" (click)="shareGist(false)"></command>
+                    <command *ngIf="isOwned|async" icon="Save" title="${Strings.updateMenu}" (click)="shareGist(false, true)"></command>
```

Comment:
please add comments to boolean (or any other non-obvious) paramaters.

e.g.,

    shareGist(false /*parameter1Name*/, true /*parameter2Name*/)